LoRaEngine/LoraKeysManagerFacade/IEdgeDeviceGetter.cs

```diff
@@ -3,11 +3,13 @@
 
 namespace LoraKeysManagerFacade
 {
+    using System.Collections.Generic;
     using System.Threading;
     using System.Threading.Tasks;
 
     public interface IEdgeDeviceGetter
     {
         Task<bool> IsEdgeDeviceAsync(string lnsId, CancellationToken cancellationToken);
+        Task<IEnumerable<string>> ListEdgeDevicesAsync(CancellationToken cancellationToken);
```

Comment:
Consider using an `ICollection` to express that this is a finite collection. If it is a streaming response, consider using `IAsyncEnumerable` instead.